conf/messages.cy

```diff
@@ -679,34 +679,37 @@ totalExpensesCYA.heading = totalExpensesCYA
 
 ########## Expenses - Repairs and Maintenance ##########
 
-########## Expenses - Repairs and Maintenance ##########
-
 ## Repairs and maintenance amount
-repairsAndMaintenanceAmount.title = How much did you spend on repairs and maintenance?
+repairsAndMaintenanceAmount.heading.individual = How much did you spend on repairs and maintenance?
+repairsAndMaintenanceAmount.heading.agent = How much did your client spend on repairs and maintenance?
 repairsAndMaintenanceAmount.details = Understanding repairs and maintenance expenses
-repairsAndMaintenanceAmount.checkYourAnswersLabel = RepairsAndMaintenanceAmountCheckYourAnswersLabel
-repairsAndMaintenanceAmount.error.nonNumeric = Enter your repairsAndMaintenanceAmount using numbers
-repairsAndMaintenanceAmount.error.required = Enter your repairsAndMaintenanceAmount
-repairsAndMaintenanceAmount.error.wholeNumber = Enter your repairsAndMaintenanceAmount using whole numbers
-repairsAndMaintenanceAmount.error.outOfRange = RepairsAndMaintenanceAmount must be between {0} and {1}
+repairsAndMaintenanceAmount.checkYourAnswersLabel.individual = How much did you spend on repairs and maintenance?
+repairsAndMaintenanceAmount.checkYourAnswersLabel.agent = How much did your client spend on repairs and maintenance?
+repairsAndMaintenanceAmount.error.required.individual = Enter the amount you spent on repairs and maintenance
+repairsAndMaintenanceAmount.error.required.agent = Enter the amount your client spent on repairs and maintenance
+repairsAndMaintenanceAmount.error.nonNumeric.individual = The amount you spent on repairs and maintenance must be a number
+repairsAndMaintenanceAmount.error.nonNumeric.agent = The amount your client spent on repairs and maintenance must be a number
+repairsAndMaintenanceAmount.error.overMax.individual = The amount you spent on repairs and maintenance must be less than £100,000,000,000
+repairsAndMaintenanceAmount.error.overMax.agent = The amount your client spent on repairs and maintenance must be less than  £100,000,000,000
+repairsAndMaintenanceAmount.error.lessThanZero.individual = The amount you spent on repairs and maintenance must be more than £0
+repairsAndMaintenanceAmount.error.lessThanZero.agent = The amount your client spent on repairs and maintenance must be more than £0
 repairsAndMaintenanceAmount.change.hidden = RepairsAndMaintenanceAmountChangeHidden
```

Comment:
This needs to be a sentence for screen readers e.g. 'Change how much was spent on repairs and maintenance' (keep it user neutral)

Reply:
nice, I was wonder how we use it and forgot to ask